src/diff/index.js

```diff
@@ -212,6 +212,10 @@ function diffElementNodes(dom, newVNode, oldVNode, context, isSvg, excessDomChil
 
 	if (newVNode.type===null) {
 		if (oldProps !== newProps) {
+			if (excessDomChildren!=null) {
+				const index = excessDomChildren.indexOf(dom);
+				if (index > -1) excessDomChildren[index] = null;
```

Comment:
We could, maybe, just directly do `excessDomChildren[excessDomChildren.indexOf(dom)] = null;`. Would this improve the size in any way? This will end-up with a property on the `excessDomChildren["-1"]` but maybe we could live with that?